testing/cmocka/Kconfig

```diff
@@ -12,6 +12,15 @@ config TESTING_CMOCKA
 
 if TESTING_CMOCKA
 
+config TESTING_CMOCKA_ASLIB
```

Comment:
it's better to use TESTING_CMOCKA to enable the library, and use TESTING_CMOCKA_PROG to enable the program.